pkgs/by-name/ff/fflogs/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, appimageTools
+, fetchurl
+}:
+
+let
+  pname = "fflogs";
+  version = "8.3.42";
+  src = fetchurl {
+    url = "https://github.com/RPGLogs/Uploaders-fflogs/releases/download/v${version}/fflogs-v${version}.AppImage";
+    sha256 = "08mkixjpppijssmg63l542xhmxyr7xs9z1v9qfjda7cfqgah5kh5";
+  };
+  extracted = appimageTools.extractType2 { inherit pname version src; };
+in
+appimageTools.wrapType2 {
+  inherit pname version src;
+
+  extraInstallCommands = ''
+    mkdir -p $out/share/applications
+    cp -r ${extracted}/usr/share/icons $out/share/
```

Comment:
please move `share/icons/hicolor/0x0/apps/fflogs.png` to `share/icons/hicolor/512x512/apps/fflogs.png`